chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -1608,15 +1608,24 @@ Zotero.Utilities.Internal = {
 	 *
 	 * @param {Library|Collection} libraryOrCollection
 	 * @param {Node<menupopup>} elem Parent element
-	 * @param {Zotero.Library|Zotero.Collection} currentTarget Currently selected item (displays as checked)
-	 * @param {Function} clickAction function to execute on clicking the menuitem.
+	 * @param {Object} options
+	 * @param {Zotero.Library|Zotero.Collection} options.currentTarget Currently selected item (displays as checked)
+	 * @param {Function} options.clickAction function to execute on clicking the menuitem.
 	 * 		Receives the event and libraryOrCollection for given item.
-	 * @param {Function} disabledPred If provided, called on each library/collection
+	 * @param {Function} options.disabledPred If provided, called on each library/collection
```

Comment:
Since this is exposed now, can we use a clearer name? `disabledPredicate`? Or a less-technical name? `isDisabledCheck`? `isDisabledFn`?

Reply:
Makes sense! We use `disabledCheck` phrasing in `dataObjects.js`, so I went with `disabledCheck` here as well. How does that look?